src/app/payment/services/checkout.service.ts

```diff
@@ -102,17 +103,20 @@ const getClientSecretForSubscriptionIntent = async ({
   priceId: string;
   token: string;
   mobileToken: string | null;
+  websiteToken: string | null;
   currency: string;
   seatsForBusinessSubscription?: number;
   promoCodeId?: string;
 }): Promise<ClientSecretData & { subscriptionId?: string; paymentIntentId?: string }> => {
-  if (mobileToken) {
+  const trialToken = mobileToken || websiteToken;
```

Comment:
Prefer to use nullish coalescing operator `??` instead of a logical or `||`. 
Here have [why](https://typescript-eslint.io/rules/prefer-nullish-coalescing/#:~:text=Because%20the%20nullish%20coalescing%20operator,An%20%7C%7C%20operator%20with%20%3F%3F)